### PR TITLE
feat: lint PR name

### PR DESCRIPTION
**Pull request by paulgrym**
## Jira issue

JIRA issue: [FSU-72](https://klau.atlassian.net/browse/FSU-72)

## Describe your changes
PR name should follow the rules from https://www.conventionalcommits.org/

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: .github/workflows/pr-name.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  pull_request:
+    types: ["opened", "edited", "reopened", "synchronize"]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Dependencies
+        run: yarn --frozen-lockfile
```
Comment (paulgrym) [inline .github/workflows/pr-name.yml:12]: This is probably too big installation, but I haven't come up with a better idea yet 😅 I will think about it later

Comment (im-klau) [reply .github/workflows/pr-name.yml:12]: yeah goog point 👍 I guess we don't need to install all the dependencies, can we just use the example from the tool you used?
`run: npm install @commitlint/config-conventional`? 🤔 

I mean installing only the `commitlint` library that's required in the `pull-request-name-linter-action`

Comment (im-klau) [reply .github/workflows/pr-name.yml:12]: btw I changed the PR title just to check the new github action 😅

Comment (paulgrym) [reply .github/workflows/pr-name.yml]: There is some dependency conflict when I use command from example 😢

Comment (im-klau) [reply .github/workflows/pr-name.yml]: I'd try the `--legacy-peer-deps` command, it's just about pull requests so it shouldn't break our code 🙈 

Zrzut ekranu 2022-11-14 o 20 50 41

**Comment by nata-kab** [comment]
@paulgrym Have you seen this warning? 🤔 
Screenshot 2022-11-18 at 13 47 11

**Comment by nata-kab** [comment]
Could you describe our rules for naming PR? 😄 To make it easier for us 😊

File: .github/workflows/pr-name.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: pr-name-linter
```
Comment (im-klau) [inline .github/workflows/pr-name.yml:1]: All looks good now 👏 

However, I've just tried caching installed dependencies to speed up the github action process and it worked 🎉 
The time has changed from about 3min to 1min.

You can check out my draft PR: https://github.com/Fundacja-Sterczace-Uszy/website/pull/49

Let me know what you think, if it's good enough then feel free to merge my PR into yours 🤗

**Review by im-klau — APPROVED** [review]

File: .github/workflows/pr-name.yml (hunk 2 above)
Comment (paulgrym) [reply .github/workflows/pr-name.yml:1]: This is a great solution, I didn't know --prefer-offline flag :)

Comment (im-klau) [reply .github/workflows/pr-name.yml:1]: Me too 😅 

I guess this PR is almost ready to be merged 🎉 just one missing approval 👀

**Pull request merged**